Profiles - ZIB 2017/conceptmap-PositieCodelijst-to-LOINC.xml

```diff
@@ -0,0 +1,45 @@
+<ConceptMap xmlns="http://hl7.org/fhir" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hl7.org/fhir http://hl7.org/fhir/STU3/conceptmap.xsd">
+      <id value="PositieCodelijst-to-LOINC"/>
+      <url value="http://nictiz.nl/fhir/ConceptMap/PositieCodelijst-to-LOINC"/>
+      <version value="1.0.1"/>
+      <name value="PositieCodelijst-to-LOINC"/>
+      <title value="HCIM BodyHeight to LOINC"/>
+      <status value="active"/>
+      <publisher value="Nictiz"/>
+      <contact>
+            <name value="Nictiz"/>
+            <telecom>
+                  <system value="email"/>
+                  <value value="info@nictiz.nl"/>
+                  <use value="work"/>
+            </telecom>
+      </contact>
+      <description value="Maps the ZIB BodyHeight release 2017 terminology for the concept PositionCodelist to the Logical Observation Identifiers Names and Codes (LOINC)"/>
+      <purpose value="To aid in mapping from ZIB BodyHeight value set PositionCodelist to LOINC"/>
+      <copyright value="CC0"/>
+      <sourceUri value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.12.2.1--20171231000000"/>
+      <targetUri value="http://loinc.org"/>
```

Comment:
I think `.target` should be removed here. There's no target ValueSet to map to, just a code system.

Reply:
Check, I've removed the .targetURI.